packages/calcite-components/src/components/input-time-zone/utils.ts

```diff
@@ -251,6 +251,12 @@ function getTimeZoneShortOffset(
   locale: SupportedLocale,
   referenceDateInMs: number = Date.now(),
 ): string {
+  // workaround for https://issues.chromium.org/issues/381620359
+  // see https://github.com/Esri/calcite-design-system/issues/10895 for more info
+  if (timeZone === "Factory") {
```

Comment:
🏭

Reply:
`"Etc/Unknown"` might be another oddity worth keeping an eye on.

![image](https://github.com/user-attachments/assets/2c79ada4-e39a-4ee8-aa52-a259be4f8825)